web/tools/webpack.config.ts

```diff
@@ -87,7 +87,7 @@ const createConfig = (
   // eslint-disable-next-line no-console
   console.log('Used config: ', buildConfigName)
   // eslint-disable-next-line no-console
-  console.log('Configured as running in dev server: ', !devServer)
+  console.log('Configured as running in dev server: ', devServer)
```

Comment:
Why this change?

Reply:
Because it was wrong before, it said `Configured as running in dev server: true` for production builds.